src/content/home.json

```
@@ -97,5 +97,20 @@
       }
     ]
   },
-  "contact": {}
+  "footer": {
+    "our_contact_label": "Our contact",
+    "call_label": "Call",
+    "email_label": "Email",
+    "business_hours_label": "Business Hours",
+    "monday_saturday": "Monday to Saturday",
+    "8am_8pm": "8:00am to 8:00pm",
+    "sundays": "Sundays",
+    "12noon_8pm": "12:00noon to 8:00pm",
```

try a more generic and flexible format. This way, it's easier to modify schedule in the future

for example

```suggestion
    "businessHours": [
      {
        "day": "Monday to Saturday",
        "time": "8:00 am to 8:00 pm"
      },
      {
        "day": "Sunday",
        "time": "12:00 noon to 8:00 pm"
      }
    ],
```